spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,95 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.bs
```

Comment:
I think this is meant to be a location where you can see the published version, ergo the html version

Reply:
updated to .html. The link will be broken for now.